src/aks-preview/azext_aks_preview/_params.py

```diff
@@ -1823,6 +1826,29 @@ def load_arguments(self, _):
             arg_type=get_enum_type(node_os_skus_update),
             validator=validate_os_sku,
         )
+        # local dns
+        c.argument(
+            "set_localdns",
+            options_list=["--set-localdns"],
+            action="store_true",
+            help="Update local DNS profile for this nodepool. Must be used with --localdns-mode or --localdns-config."
+        )
+        c.argument(
+            "localdns_config",
+            options_list=["--localdns-config"],
+            type=file_type,
```

Comment:
You may omit this since the default type is `str`, which I believe is appropriate for representing a file path.

Reply:
Fixed this, thank you!